### ISSUE TITLE
Suicide Vest#3

### ISSUE THREAD

**Issue by WarlordX07** [71stSOG/71stSOG-Insurgency#14]
Suicide Vest reloads and can add multiple add action options to use multiple vest.

**Comment by onedigita** [comment]
yes this need to be limited some how not that it makes a huge difference but this could cause some lag im wondering
